examples/nemo/scripts/Makefile

```diff
@@ -0,0 +1,108 @@
+# Makefile to Generate PSyclone versions of NEMO, Compile, and Run them.
+# It supports the targets: serial (only compile), openmp-cpu, openmp-gpu, openacc-kernels
+# - To generate the psycloned versions use: `make <target>` 
+# - To compile that version use: `make compile-<target>`
+# - To run that version use: `make run-<target>`
+
+# ---- Start of the configurable part of the Makefile ----
+
+# - Specify NEMO directory
+# NEMO_DIR ?= /home/ssiso/workspace/stfc/NEMO
```

Comment:
I now use a variable, WORKSPACE instead of HOME, so it can reference any starting prefix.

However, I still left the 2 MODEL_DIR full paths because this are in a different user's directory for now. This are large files and Glados is quite full. Should I replicate those inside gh_runner?

Reply:
Oh yes. I think we can leave it for now and solve it when we address the perl location and/or portability of the whole suite.